src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,33 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.info("Method completeOrder called. Params: userId={}", userId);
         List<Product> products = getAllProductsFromShoppingCart(userId);
         Order order = new Order(products, userId);
         // NOTE: In production ready code this order identifier should be generated by DB
         // For test purpose we simplify this and return dummy data
         order.setOrderId(1L);
+        logger.debug("Order completed: {}", order.getOrderId());
         return order;
     }
 
     private List<Product> getAllProductsFromShoppingCart(Long userId) {
         // NOTE: In production ready code this method should fetch data from DB
         // For test purpose we simplify this method and return dummy data
+        logger.info("Fetching products for userId={}", userId);
```

Comment:
The log message 'Fetching products for userId={}' is informative and provides context for the dummy data being used.